### PR TITLE
Improve exception messages when there is an error unsseting an agent from a group

### LINKED ISSUE #2826: Errors trying to delete agents of their last group
Hi team,

I found a couple of errors trying to replicate the error cases of this issue https://github.com/wazuh/wazuh-api/issues/346.

When I try to remove a single agent of its last group using this query:
```js
DELETE /agents/001/group/default
```
The error message is correct, but the error code is 0, so we can not differentiate if there has been success or not.

```js
{
  "error":0,
  "data":"Agent only belongs to 'default' and it cannot be unset from this group."
}
```

When I try to remove agents of their last group using the app's multiple agents selector and this query:

```js
DELETE /agents/group/default
{
 "ids": ["001", "002"] 
}
```

I receive the following output:
```js
{
  "error": 0,
  "data": {
    "msg": "All selected agents were removed to group default",
    "affected_agents": [ "001","002" ]
  }
}
```

When they haven't really been deleted.

Regards.

### PR DESCRIPTION
**Pull request by mgmacias95**
Hello team,

This PR fixes #2826.

Some examples:

Removing an agent from the default group:
```javascript
# curl -u foo:bar "localhost:55000/agents/001/group/default?pretty" -XDELETE
{
   "error": 1745,
   "message": "Agent only belongs to 'default' and it cannot be unset from this group."
}
```

Removing multiple agents from the default group. The format of this function is now the same as the `POST/agents/restart` and `POST/agents/delete`:
```javascript
# curl -u foo:bar -X DELETE -H "Content-Type:application/json" -d '{"ids":["001"]}' "http://localhost:55000/agents/group/default?pretty"
{
   "error": 0,
   "data": {
      "msg": "Some agents were not removed from group default",
      "affected_agents": [],
      "failed_ids": [
         {
            "id": "001",
            "error": {
               "message": "Agent only belongs to 'default' and it cannot be unset from this group.",
               "code": 1745
            }
         }
      ]
   }
}
```

Removing an agent from a group it doesn't belong to:
```javascript
# curl -u foo:bar "localhost:55000/agents/001/group/defaulffdft?pretty" -XDELETE
{
   "error": 1734,
   "message": "Agent does not belong to the specified group"
}
```

Removing an agent from a group when it doesn't belong to any group at all:
```javascript
# curl -u foo:bar "localhost:55000/agents/002/group/defaulffdft?pretty" -XDELETE
{
   "error": 1734,
   "message": "Agent does not belong to the specified group"
}
```

Unit tests implemented:
```python
# /var/ossec/framework/python/bin/pytest tests/test_group.py -vv
============================================================================================================================ test session starts ============================================================================================================================
platform linux -- Python 3.7.2, pytest-4.3.1, py-1.8.0, pluggy-0.9.0 -- /var/ossec/framework/python/bin/python3.7
cachedir: .pytest_cache
rootdir: /home/vagrant/GitHub/wazuh/framework, inifile:
collected 8 items

tests/test_group.py::test_sucessfully_remove_single_group_agent[dmz-005-dmz-default] PASSED                                                                                                                                                                           [ 12%]
tests/test_group.py::test_sucessfully_remove_single_group_agent[dmz,webserver-005-dmz-webserver] PASSED                                                                                                                                                               [ 25%]
tests/test_group.py::test_sucessfully_remove_single_group_agent[dmz,webserver,database-005-dmz-webserver,database] PASSED                                                                                                                                             [ 37%]
tests/test_group.py::test_sucessfully_remove_single_group_agent[dmz,default-005-default-dmz] PASSED                                                                                                                                                                   [ 50%]
tests/test_group.py::test_failed_remove_single_group_agent[-005-dmz-1734] PASSED                                                                                                                                                                                      [ 62%]
tests/test_group.py::test_failed_remove_single_group_agent[dmz-005-default-1734] PASSED                                                                                                                                                                               [ 75%]
tests/test_group.py::test_failed_remove_single_group_agent[default-005-default-1745] PASSED                                                                                                                                                                           [ 87%]
tests/test_group.py::test_failed_remove_single_group_agent[dmz-005-webserver,database-1734] PASSED                                                                                                                                                                    [100%]

========================================================================================================================= 8 passed in 0.31 seconds ==========================================================================================================================
```

Best regards,
Marta

### CODE REVIEW

File: framework/wazuh/agent.py [hunk 1]
```diff
@@ -1680,24 +1680,21 @@ def unset_group_list(group_id, agent_id_list):
         if not Agent.group_exists(group_id):
             raise WazuhException(1710)
 
+        message = f'All selected agents were removed to group {group_id}'
```
Comment (crd1985) [inline framework/wazuh/agent.py]:
```suggestion
        message = f'All selected agents were removed from group {group_id}'
```

File: framework/wazuh/exception.py [hunk 2]
```diff
@@ -138,6 +138,8 @@ class WazuhException(Exception):
         1742: 'Error running XML syntax validator',
         1743: 'Error running Wazuh syntax validator',
         1744: 'Invalid chunk size',
+        1745: "Agent only belongs to 'default' and it cannot be unset from this group.",
+        1746: "Agent doesn't belong to any group",
```
Comment (crd1985) [inline framework/wazuh/exception.py]: I have the doubt between these two messages:
* Agent doesn't belong to any group
* Agent belongs to default group only and cannot be removed

Comment (mgmacias95) [reply framework/wazuh/exception.py]: The exception `1746`  is raised when the agent doesn't belong to any group, it's the first check the function does.
The exception `1745` is raised when the function receives `default` as `group_id` argument. This exception is raised when the agent belongs to any group.

Comment (mgmacias95) [reply framework/wazuh/exception.py]: I have finally removed exception `1746` in commit https://github.com/wazuh/wazuh/pull/2838/commits/2ddf6b7e6c6a0df82fab94dedfbb85b4c7abe731.

**Pull request merged**